File: Programming-Basics-with-Python/9.While Loop - Lab/02. Password.py
username = input()
password = input()

# повтаряме: въвеждаме парола
# спираме: въведената парола == password
# продължаваме: въведенета парола != password
entered_password = input()

while entered_password != password:
    entered_password = input()
else: # entered_password == password
    print(f"Welcome {username}!")

File: Programming-Basics-with-JavaScript/5.While Loop/02password.js
// function passwords(input){
//     let urerName = input[0]
//     let password = input[1]
//     let data = input[2]
//     let idx = 3
//     while (data !== password){
        
//         data = input[idx]
//         idx++

//     }
//     console.log(`Welcome ${urerName}!`)

// }
// passwords(["Nakov",
// "1234",
// "Pass",
// "1324",
// "1234"])
function passwords(input){
    let idx = 0 
    let urerName = input[idx]
    idx++
    let password = input[idx]
    idx++
    let data = input[idx]
    idx++
    while (data !== password){
        
        data = input[idx]
        idx++

    }
    console.log(`Welcome ${urerName}!`)

}
passwords(["Nakov",
"1234",
"Pass",
"1324",
"1234"])